dot/parachain/collator-protocol/validator_side.go

```diff
@@ -0,0 +1,316 @@
+// Copyright 2023 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package collatorprotocol
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/ChainSafe/gossamer/dot/network"
+	parachaintypes "github.com/ChainSafe/gossamer/dot/parachain/types"
+	"github.com/ChainSafe/gossamer/dot/peerset"
+	"github.com/ChainSafe/gossamer/internal/log"
+	"github.com/ChainSafe/gossamer/lib/common"
+	"github.com/libp2p/go-libp2p/core/peer"
+	"github.com/libp2p/go-libp2p/core/protocol"
+	"golang.org/x/exp/slices"
+)
+
+var logger = log.NewFromGlobal(log.AddContext("pkg", "collator-protocol"))
+
+const (
+	ACTIVITY_POLL              = 10 * time.Millisecond
+	CHECK_COLLATIONS_POLL      = 50 * time.Millisecond
+	MAX_UNSHARED_DOWNLOAD_TIME = 100 * time.Millisecond
```

Comment:
Just a nit for code consistency

```suggestion
	ActivityPool = 10 * time.Millisecond
	CheckCollationsPoll = 50 * time.Millisecond
	MaxUnsharedDownloadTime = 100 * time.Millisecond
```